.github/workflows/pr-close-delete-env.yml

```diff
@@ -5,25 +5,61 @@ on:
     types: [closed]
   workflow_dispatch:
     inputs:
-      environment:
-        description: "Pull request number"
+      slotID:
+        description: "Pull request number (Add multiple IDs seperated by a space)"
         required: true
         default: ""
+  workflow_call:
+    inputs:
+      slotIDs:
+        type: string
+        description: "Slot ID (Add multiple IDs seperated by a space)"
+        required: true
 
 defaults:
   run:
     shell: pwsh
 
 env:
-  SLOT_NAME: pr-${{ github.event.number }}
-
+  SLOT_NAME: ${{ inputs.slotIDs || inputs.slotID || github.event.number }}
+  SLOT_PREFIX: pr-
 permissions:
   id-token: write
   contents: read
 
 jobs:
+  setting-up-slot-ids:
```

Comment:
add a step above this one, it should be run only if the github.event is pull_request - it should format the SLOT_NAME as the event number in a json array

Reply:
Instead of checking github.event is pull request, the current step can handle all the input 
<img width="568" alt="image" src="https://github.com/SSWConsulting/SSW.Website/assets/71385247/7a17b175-d32f-498a-a543-14c485fc9ddd">
**Figure: it converts the input and then format it accordingly.**